tests/python/gaia-ui-tests/gaiatest/apps/search/app.py

```diff
@@ -9,7 +9,6 @@
 class Search(Base):
 
     name = 'Browser'
-    manifest_url = "app://search.gaiamobile.org/manifest.webapp"
```

Comment:
So this just works?
So the name is used? But the name is different from what there is in the manifest_url, namely 'search'.